### PR TITLE
Create ui-c8y-1019-22-1-btn-default-for-sso-login-button-5488-graft-release-cd-6158.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-57675](https://cumulocity.atlassian.net/browse/MTM-57675)
Original PR: [6158](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6158)

### CODE REVIEW

File: content/change-logs/platform-services/ui-c8y-1019-22-1-btn-default-for-sso-login-button-5488-graft-release-cd-6158.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: btn-default for sso login button (#5488) [GRAFT][release/cd] (#6158)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1019-22-1-btn-default-for-sso-login-button-5488-graft-release-cd-6158.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Improved styling of single sign-on login button
```

File: content/change-logs/platform-services/ui-c8y-1019-22-1-btn-default-for-sso-login-button-5488-graft-release-cd-6158.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: q3kclF6pO
+    label: Authentication
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57675
+version: 1019.22.1
+---
+btn-default for sso login button (#5488) [GRAFT][release/cd] (#6158)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1019-22-1-btn-default-for-sso-login-button-5488-graft-release-cd-6158.md:17]: The Claude AI suggested to use the following body:
```suggestion
To provide a more consistent user experience, the styling of the single sign-on (SSO) login button has been updated. Previously, the SSO login button used a custom style which did not match the overall look and feel of the application. With this change, the SSO login button now uses the default button style (btn-default). This improvement in styling ensures that the SSO login button integrates seamlessly with the rest of the user interface, making it easier for users to recognize and interact with the SSO functionality.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by BeateRixen** [comment]
@kalavathig1 @sarangovindarajan Please review and approve.

**Review by mmed-softwareag — APPROVED** [review]

**Review by pink-mongoose — CHANGES_REQUESTED** [review]
We should remove the use of the word 'style' this has no meaning to a customer.  Perhaps something like:

The user experience for the single sign-on (SSO) has been improved with an updated login button to match the overall look and feel of the application.

**Review by BeateRixen — APPROVED** [review]

File: content/change-logs/platform-services/ui-c8y-1019-22-1-btn-default-for-sso-login-button-5488-graft-release-cd-6158.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Improved styling of single sign-on login button
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1019-22-1-btn-default-for-sso-login-button-5488-graft-release-cd-6158.md:3]:
```suggestion
title: Improved layout of single sign-on login button
```

File: content/change-logs/platform-services/ui-c8y-1019-22-1-btn-default-for-sso-login-button-5488-graft-release-cd-6158.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: q3kclF6pO
+    label: Authentication
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57675
+version: 1019.22.1
+---
+The styling of the single sign-on (SSO) login button has been updated. Previously, the SSO login button used a custom style which did not match the overall look and feel of the application. With this change, the SSO login button now uses the default button style (btn-default).
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1019-22-1-btn-default-for-sso-login-button-5488-graft-release-cd-6158.md:17]:
```suggestion
The layout of the single sign-on (SSO) login button has been updated. Previously, the SSO login button used a custom style which did not match the overall look and feel of the application. With this change, the SSO login button now uses the default button style (btn-default).
```

**Comment by BeateRixen** [comment]
> We should remove the use of the word 'style' this has no meaning to a customer. Perhaps something like:
> 
> The user experience for the single sign-on (SSO) has been improved with an updated login button to match the overall look and feel of the application.

@pink-mongoose How about using the term "layout"? This is a term we often use. "User expericence" seems a bit too much when talking about a simple button change, isn´t it :-)?

File: content/change-logs/platform-services/ui-c8y-1019-22-1-btn-default-for-sso-login-button-5488-graft-release-cd-6158.md (hunk 4 above)
Comment (pink-mongoose) [reply content/change-logs/platform-services/ui-c8y-1019-22-1-btn-default-for-sso-login-button-5488-graft-release-cd-6158.md:17]: @BeateRixen  I still do not like the fact we are using the default button style.  The last sentence can be completely removed.

**Review by kalavathig1 — APPROVED** [review]

File: content/change-logs/platform-services/ui-c8y-1019-22-1-btn-default-for-sso-login-button-5488-graft-release-cd-6158.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: q3kclF6pO
+    label: Authentication
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57675
+version: 1019.22.1
+---
+The layout of the single sign-on (SSO) login button has been updated. Previously, the SSO login button used a custom style which did not match the overall look and feel of the application. With this change, the SSO login button now uses the default button style (btn-default).
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1019-22-1-btn-default-for-sso-login-button-5488-graft-release-cd-6158.md:17]:
```suggestion
The layout of the single sign-on (SSO) login button has been updated. Previously, the SSO login button used a custom style which did not match the overall look and feel of the application. With this change, the SSO login button now uses the default layout.
```

**Pull request merged**